### ISSUE TITLE
feature request: option to kill and detach from session

### ISSUE THREAD

**Issue by marcoSven** [joshmedeski/t-smart-tmux-session-manager#30]
Would be nice to be able to add an option to kill and detach from a session.

I am currently using this

```bash

temp_session=$(printf '%s\n' "${$(for ip in "${$(tmux list-sessions -F "#{session_name}")[@]}"; do echo "${ip}"; done | sort -u)[@]}" | fzf-tmux --reverse -p 70% --border-label "Kill TMUX Session") 

test -n "$temp_session" && tmux detach -s $temp_session && tmux kill-session -t $temp_session


```

**Comment by joshmedeski** [comment]
Do you have a preference on what keyboard shortcut to use? I've been playing around with `ctrl+x` to kill a session from the t popup.

**Comment by marcoSven** [comment]
`ctrl+x` sounds great. 

Looks like we are already using that binding to filter for `zoxide` but maybe that can me changed to `ctrl+z`

**Comment by marcoSven** [comment]
I left a review comment. How about adding a binding to detach in addition to kill?

**Comment by joshmedeski** [comment]
Why would you want to detach? I guess I'm not understanding the feature. If the session is being killed there's no need to do a detach first.

**Comment by marcoSven** [comment]
Occasionally I detach rather than kill. Just think it would be nice to add this to the session manager. 

"^K" kills the session "^d" detaches from the session 

Frankly I mostly use an alias to detach "td". But sometimes I do use this script to have an interactive selection.

**Comment by marcoSven** [comment]
Thank you, I decided to create my own little script https://github.com/marcoSven/tmux-session-manager which allows me to kill and detach from a session. 

I ended up using "^x" to kill and "^d" to detach. "^z" is obviously Zoxide filter and I use "^p" to to filter for directories which is consistent with my other shortcuts. 

<img width="739" alt="image" src="https://github.com/joshmedeski/t-smart-tmux-session-manager/assets/3860298/4a8d4995-0163-4c26-be80-241a1f7aebc7">

I don't see the harm to add an additional binding it allows me to remove all my other scripts.

Feel free to close the issue.
